gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -68,6 +71,8 @@ export class AdvancedDynamicTexture extends DynamicTexture {
     private _blockNextFocusCheck = false;
     private _renderScale = 1;
     private _rootCanvas: Nullable<HTMLCanvasElement>;
+    private _clipboardData: DataTransfer;
+    public onClipboardObserver = new Observable<ClipboardInfo>();

```

Comment:
"onClipboardObservable"

Reply:
my bad 😿 , will fix it right away.. 
I was playing with observer and observable must have interchanged them at end

Reply:
no worry :)